### ISSUE TITLE
Final Peer Review By sz244

### ISSUE THREAD

**Issue by sz244** [vanshikabansal/ORIE-4741#13]
Things that I like about the project:
1. I like  there feature selection process, especially how they deal with historical data in the test set, they are worried that including that would skew the result so  they fill in the median value of the historical data of the horse. I think it's good choice, better than doing matrix completion because it would be difficult to derive historical statistic from only the weight/date information. On the other hand, I think they can try retaining the rows with missing data and throwing new horses to see how large having new horse would affect the result. Because if they notice that their model heavily relies on the historical data, then it would be bound to have problematic predictions on missing data horses. 

2. They kept an close eye to the test error and training set error and overfitting issue.

3.  The have realistic interpretation of their result and errors and was able to identify that historical data is the main driver in predictions.

Some concerns/Suggestions.
1. It's a great choice to try to not let models depend on specific horses.  But I'm slightly worried that if the size of dataset is small(the project report did not specify how many datapoints are there), then the weight of the horse and historical statistic might easily serve as a identifier for the horse and cause model to overfit. Since their test set will probably contains horses seen in the training set(since the split is random), the prediction accuracy would be higher than if we use only new horses.   I think it would be interesting to have more analysis on that part as suggested in part 1 of the section above.

2. The graphs are quite difficult to understand. Especially the actual-predicted percentile graph.

3. It would be better if they explain or provide some intuition on why did L1 perform much better(the prediction quantile actually showed some trend that correlate with the actual quantile) then the Huber loss function even though they are both robust loss function not so sensitive to outliers.
